qt/applications/workbench/workbench/plotting/figuremanager.py

```diff
@@ -193,22 +192,6 @@ def __init__(self, canvas, num):
         self.window.setWindowTitle("Figure %d" % num)
         canvas.figure.set_label("Figure %d" % num)
 
-        # Set tight layout so axes labels remain visible
```

Comment:
When this is looked at again perhaps we could add a unit test that the warning `This figure includes Axes that are not compatible with tight_layout, so results might be incorrect` is not produced when creating common types of plot?

Reply:
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertNoLogs might help with this...